topdown/topdown_test.go

```diff
@@ -3015,7 +3015,7 @@ func assertTopDownWithPathAndContext(ctx context.Context, t *testing.T, compiler
 				t.Fatal(err)
 			}
 
-			expected := util.MustUnmarshalJSON([]byte(e))
+			expected = util.MustUnmarshalJSON([]byte(e))
```

Comment:
Lets revert this line and keep it using `:=` instead of `=`, no need to change it.